### DISCUSSION TITLE
Azure template

### DISCUSSION THREAD

**Discussion by estimpfle** [microsoft/finops-toolkit#1110] [Q&A] [answered]
Hello.  I found the Microsoft documentation for Azure.  It appears that we have to scope the export to a single subscription and says that it is not available for management groups.  We need to be able to scope it to our root management group or to the billing invoice ID. It says that we have a limit of 10 exports. And in my organization, we have 30 subscriptions.   Is there anyone who has experience around this issue?  We would like to able to use the built-in export template but for most organizations scoping it to single subscriptions is not going to be a very good option.

**Comment by flanakin** [comment, marked as answer]
We recommend creating exports at the billing account or billing profile scope, which will cover all subscriptions and also add support for additional datasets, like prices and reservation data. 

The limit of 10 exports is only in the create form. You can create more than 10 exports per scope. In your case, you're talking about creating subscription-scoped exports, which would be on each of the 30 subscriptions, which would require changing scope for each if using the portal to create them manually. Consider using the [`New-FinOpsCostExport` command](https://aka.ms/ftk/New-FinOpsCostExport) to create exports at scale.

We have discussed adding support for automating subscription-level exports. If billing accounts/profile exports aren't available and management group automation would be useful, please submit a feature request to add it to the backlog.
